### DISCUSSION TITLE
How to make mini map transparent and acrylic?

### DISCUSSION THREAD

**Discussion by prathmesh-ka-github** [illixion/vscode-vibrancy-continued#137] [Q&A] [answered]
Pls help, the mini map is looking terrible as it is the only thing that has the theme color.
Thanks.

**Comment by illixion** [comment]
Which theme are you using?

**Reply by prathmesh-ka-github** [reply]
I am using the theme Tomorrow Night Blue.

**Reply by illixion** [reply, marked as answer]
Please switch to one of the supported themes listed here: https://github.com/illixion/vscode-vibrancy-continued?tab=readme-ov-file#vscode_vibrancytheme
